### ISSUE TITLE
@neo4j/graphql-ogm v6 release missing

### ISSUE THREAD

**Issue by to-kn** [neo4j/graphql#5756]
**Describe the bug**
Hi, as the title says, we are missing the v6 release of @neo4j/graphql-ogm package, is there any eta?
Thanks a lot! Best regards Tobias

**Comment by darrellwarde** [comment]
Hi @to-kn, there is no plan to release the OGM for Version 6 and Version 7 of the library at this time. You can find some more details in my recent blog: https://neo4j.com/developer-blog/neo4j-graphql-library-roadmap/

As detailed in that blog, we will be maintaining Version 5 of the library as an LTS along with the OGM, we recommend sticking on Version 5 if you need the OGM.

**Comment by to-kn** [comment]
Hi @darrellwarde , thank you for the clarification and info! Do i get it right, that instead of the ogm we/one has to use directly the session/cypher queries in the future?

**Comment by darrellwarde** [comment]
We want to look at the OGM from a more strategic, company-wide standpoint. I hate to say it but the OGM in its current state is a bit of a hack-job, and ideally it is replaced by something doing native Cypher translation rather than using GraphQL under the hood. For now, stick with Version 5 if you need the OGM, and keep your eyes and ears open for updates in the future!

**Comment by Abhid14** [comment]
Greetings @darrellwarde 

Kudos to the entire Neo4j team and contributors for bringing the power of graphs to developers' fingertips. At our team, we use Neo4j 5.0 in production, and the OGM really delivers an S-tier developer experience - everyone in our team truly loves using it to power our Next.js full-stack application.

We developed an automatic type generation script that generates TypeScript definitions from the GraphQL schema in our codebase, which we built following the old documentation. This gives us full confidence in our queries and provides an excellent developer experience, even better than ORMs like Prisma for querying relational data on Postgres.

We are both excited and slightly sad about the GraphQL changes coming to Neo4j - sad only about saying goodbye to the OGM library for upcoming releases. We've never had as much fun working with a database as we have with Neo4j, thanks to Cypher and particularly the OGM library, which provides an ORM-like experience for our graph.

**I have two questions:**

1. Are there any plans to bring back the OGM experience or provide an alternative library in upcoming releases?
2. If OGM doesn't make it back on the roadmap, are there any workarounds to achieve that type safety or similar experience using one or more libraries to stay up to date with the latest versions of Neo4j?
